cylc/flow/network/schema.py

```diff
@@ -283,22 +277,35 @@ class SortArgs(InputObjectType):
 
 # Resolvers:
 
-def get_type_str(obj_type):
-    """Iterate through the objects of_type to find the inner-most type."""
-    pointer = obj_type
-    while hasattr(pointer, 'of_type'):
-        pointer = pointer.of_type
-    return str(pointer).replace('!', '')
```

Comment:
`graphql-core` comes with a function that does this https://github.com/graphql-python/graphql-core-legacy/blob/v2.3.2/graphql/type/definition.py#L54-L60